core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -1376,11 +1376,29 @@ public boolean hasSubjects(String subject,
   @Override
   public Schema getLatestVersion(String subject) throws SchemaRegistryException {
     try (CloseableIterator<SchemaRegistryValue> allVersions = allVersions(subject, false)) {
-      List<Schema> sortedVersions = sortSchemasByVersion(allVersions, false);
-      return sortedVersions.size() > 0 ? sortedVersions.get(sortedVersions.size() - 1) : null;
+      return getLatestVersionFromSubjectSchemas(allVersions, false);
     }
   }
 
+  private Schema getLatestVersionFromSubjectSchemas(CloseableIterator<SchemaRegistryValue> schemas,
+                                                    boolean returnDeletedSchema) {
```

Comment:
Parameter `returnDeletedSchema` does not seem needed, as it's always set to false. Do we plan on using it in the near future or can we remove it ?

Reply:
Agreed, we can remove it. I kept it there to make it consistent with all the other 'get' methods we currently have.

Reply:
Addressed.